draft/methodologies/transportation/carpool/AdditionalityService.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.4.22;
```

Comment:
the latest `truffle` only supports `0.4.21` — what are you using to compile this?